src/main/java/org/apache/sling/commons/contentdetection/internal/ContentAwareMimeTypeServiceImpl.java

```diff
@@ -52,10 +53,14 @@ public String getMimeType(String filename, InputStream content) throws IOExcepti
         if(!content.markSupported()) {
             throw new IllegalArgumentException("Supplied InputStream does not support mark/reset");
         }
-        TikaInputStream stream = TikaInputStream.get(content);
-        Metadata metadata = new Metadata();
-        metadata.set(Metadata.RESOURCE_NAME_KEY, filename);
-        MediaType mediaType = detector.detect(stream, metadata);
+        MediaType mediaType;
+        TemporaryResources tmp = new TemporaryResources();
```

Comment:
Please use try with resources and don't forget to close tmp: https://tika.apache.org/1.8/api/org/apache/tika/io/TikaInputStream.html#get(java.io.InputStream,%20org.apache.tika.io.TemporaryResources)